### PR TITLE
Modify HUD display of pod lifecycle

### PR DESCRIPTION
**Pull request by marionbarker**
Modify HUD pod lifecycle display logic for Eros pods to include user selected expiration warning.
Please see [PR 40: modify HUD pod lifecycle display](https://github.com/LoopKit/OmniBLE/pull/40) for more information about motivation and design.

Warning - The code builds onto a test phone and I think it will modify behavior as desired but I have not tested it.
I hope to find an Eros user willing to test this PR.

Two changes:
* OmniKit/Model/Pod.swift:
```
    Add timeRemainingWarningThreshold
    Remove expirationAlertWindow which is not used and has a name similar to
    expirationReminderAlertDefaultTimeBeforeExpiration which is used
```
* OmniKit/PumpManager/OmnipodPumpManager.swift:
```
modify HUD pod lifecycle display logic based on user selected expiration reminder
```

### CODE REVIEW

File: OmniKit/PumpManager/OmnipodPumpManager.swift [hunk 1]
```diff
@@ -287,18 +287,19 @@ extension OmnipodPumpManager {
         guard let podState = state.podState, let expiresAt = podState.expiresAt else {
             return nil
         }
+        let eolDisplayActiveTime = expiresAt.timeIntervalSinceNow + Pod.timeRemainingWarningThreshold
```
Comment (ps2) [inline OmniKit/PumpManager/OmnipodPumpManager.swift:290]: `expiresAt` is the nominal expiration at 72 hours. `scheduledExpirationReminder` is the time of the user scheduled expiration reminder. I think you meant to use the latter?

Comment (marionbarker) [reply OmniKit/PumpManager/OmnipodPumpManager.swift:290]: I believe OmniKit should match the OmniBLE version.  I'll hold off until the PR for OmniBLE is resolved. Then update this PR to be consistent.

Comment (marionbarker) [reply OmniKit/PumpManager/OmnipodPumpManager.swift:290]: Furthermore this branch is out of date wrt dev and needs merge conflicts resolved.

**Comment by marionbarker** [comment]
I'm closing this and will open a new one similar to the OmniBLE PR.

**Pull request closed without merging**